Lib/asyncio/coroutines.py

```diff
@@ -52,6 +53,9 @@ def __repr__(self):
         return f'<{self.__class__.__name__} {coro_repr}>'
 
     def __iter__(self):
+        warnings.warn("yield from coro is deprecated, please use await coro instead",
```

Comment:
Maybe use coroutine instead of the coro because I think `coro` is not defined in the glossary.
 
`yield from coroutine`
`await coroutine`

Reply:
Maybe.
On the other hand, I don't like too long messages.
Usually, I rely on @1st1 judgment for texts :)